code/modules/clothing/wrists/watches.dm

```diff
@@ -56,7 +56,7 @@
 	if(wired && screwed)
 		to_chat(usr, "You check your watch, spotting a digital collection of numbers reading '[worldtime2text()]'. Today's date is '[time2text(world.time, "Month DD")]. [game_year]'.")
 		if (evacuation_controller.get_status_panel_eta())
-			to_chat(usr, SPAN_WARNING("The shuttle's status is reported as: [evacuation_controller.get_status_panel_eta()]."))
+			to_chat(usr, SPAN_WARNING("Time until Bluespace Jump: [evacuation_controller.get_status_panel_eta()]."))
```

Comment:
a better solution would be to check for the current map, and if it is equal to the ship, show above, otherwise show nothing.